src/mocks/api/shopsResult.ts

```diff
@@ -0,0 +1,31 @@
+import { http, HttpResponse } from 'msw';
+import { API_PATH } from 'config/constants';
+import shopsResultData from './data/shopsResultData';
+
+interface RequestBody {
+  lat: number | undefined;
+  lng: number | undefined;
+}
+
+const handlers = [
+  http.post(`${API_PATH}/shops`, async ({ request }) => {
+    const body = await request.json() as RequestBody;
+
+    body.lat = 37.71;
+    body.lng = 127.97;
+
+    const url = new URL(request.url.toString());
+    const keyword = url.searchParams.get('keyword');
+
+    if (typeof body.lat !== 'number' || typeof body.lng !== 'number') {
+      return new HttpResponse(null, { status: 401 });
+    }
+
+    const filteredShops = shopsResultData.filter((shop) => shop.name.includes(keyword ?? ''));
+    return HttpResponse.json({
+      shopQueryResponseList: filteredShops,
+    });
```

Comment:
```
{
  "pageToken": string | null,
  "shopQueryResponseList": Shop[]
}
```
 response body 의 형태가 이러하니 return 하는 객체 형태를 위와 맞게 해주는 것이 좋을 것 같아요!

```suggestion
    return HttpResponse.json({
       pageToken: null,
      shopQueryResponseList: filteredShops,
    });
```

Reply:
수정했습니다.[1141eab](https://github.com/BCSDLab/JJBAKSA_FRONT_END/pull/117/commits/1141eab09c67ef5d62f21da7a3379c54b5333d63)